packages/interfaces/src/limetype.interface.ts

```diff
@@ -7,6 +7,15 @@ export interface Limetype {
         plural: string;
     };
     properties: Properties;
+
+    /**
+     * A function to get a property by name
```

Comment:
I don't think we need to state that this is a function :smile: 
```suggestion
     * Get a property by name
```

Reply:
@jgroth https://github.com/Lundalogik/lime-web-components/blob/ac4c3e91ddf40cb832a3d2b91c7caab9f724bc44/packages/interfaces/src/limeobject.interface.ts#L27-L37 :)

Reply:
Was just trying to follow conventions!